config/kubernetes-sigs/sig-storage/teams.yaml

```diff
@@ -94,6 +94,24 @@ teams:
     - wlan0
     - xing-yang
     privacy: closed
+  cosi-driver-minio-admins:
+    description: Admin access to cosi-driver-minio
+    members:
+    - jsafrane
+    - msau42
+    - saad-ali
+    - xing-yang
+    privacy: closed
+  cosi-driver-minioc-maintainers:
```

Comment:
This says "cosi-driver-minioc-maintainers".  Should it be "cosi-driver-minio-maintainers"?

Reply:
cc @nikhita

Reply:
> This says "cosi-driver-minioc-maintainers". Should it be "cosi-driver-minio-maintainers"?


oops, yes. created https://github.com/kubernetes/org/pull/2654 to fix this